scripts/install-dev.sh

```diff
@@ -955,7 +955,9 @@ configure_backendai() {
   # initialize the DB schema
   show_info "Setting up databases..."
   ./backend.ai mgr schema oneshot
+  ./backend.ai mgr fixture populate fixtures/manager/example-users.json
   ./backend.ai mgr fixture populate fixtures/manager/example-keypairs.json
+  ./backend.ai mgr fixture populate fixtures/manager/example-set-user-main-access-keys.json --override
```

Comment:
Even this works, it is better to keep the options in front of the arguments.
```suggestion
  ./backend.ai mgr fixture populate --override fixtures/manager/example-set-user-main-access-keys.json
```

You may experience very strange and hard-to-debug errors if you mix up options and arguments in gcc and linkers, for example. Always make your habits to conform with the standard convention.